src/css/custom.css

```diff
@@ -59,3 +59,12 @@ a {
 .navbar__brand:hover{
   color: var(--ifm-navbar-link-hover-color);
 }
+
```

Comment:
As a minor update, we could make the width of tables 100% so that the width of both code blocks and tables are the same. Just so that they look more organized.

```suggestion

.propertyTable {
  display: table;
}

```

Before:
<img width="845" alt="Screen Shot 2021-10-12 at 10 22 01 PM" src="https://user-images.githubusercontent.com/9922882/136964451-fc9e6101-f559-42c7-a849-7fc9374a5a62.png">

After:
<img width="817" alt="Screen Shot 2021-10-12 at 10 22 32 PM" src="https://user-images.githubusercontent.com/9922882/136964487-6087bbd0-40ee-4e03-88af-aa2c040ae9aa.png">

Reply:
I agree. Thanks!